communication/youtube/youtube-guidelines.md

```diff
@@ -184,7 +184,7 @@ public, and added context to the description.
 
 Example description:  
 Kubernetes Contributor Experience Special Interest Group Weekly Meeting. Check
-here (link to sig list) for a complete list of SIGs and when they meet. Join us!
+here for a [complete list of SIGs] and when they meet. Join us!
```

Comment:
As this is intended to be an example message put into the YouTube meeting description - the `(link to sig list)` is meant to be a placeholder for folks to put in their own SIG list link. It could be made clearer by putting the example message in a codeblock though. I made an example below since I can't to it as a suggestion^^;;

---
Example description:
```
Kubernetes Contributor Experience Special Interest Group Weekly Meeting. Check
here (link to sig list) for a complete list of SIGs and when they meet. Join us!	
```

Reply:
My bad, made a mistake (^_^;). Will fix it. Thanks for reviewing :).